tests/encapsed/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -121,8 +121,7 @@ $content = sprintf(
     <<<EOF
 foo
 EOF
-
-);
+    );
```

Comment:
Now the indent seems off 🙈

Reply:
ugh 😕 thought this might be how js does it but apparently [not](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEASSsEwAQF5tgCGANsQCpwDOMAFADpTZNMAGAyhALZzbzXYQAZg2LQA5tmoAnAJawWASgDcIADQgIABxgzolZKEJSpEAO4AFIwn0pCANwgyAJmpCFqyQSUpx1AIylCMABrOBg2TSC5MWQYKQBXXxAnCDBPbyS5HykYc0CxTkJ04h91ACtKAA8AIUCQsLZCbgAZOThi0o14mE1ugCYOpMipbOQQP0I-AE9RKFdNWVgAdWcYAAtkAA4ABnUFiB8lwM0xhao4KTt29Sk4AEd4mVu8wgKipC8SpJ9OGViE77RYhwACK8Qg8EG6hgkxWTnWyD60MCMmI0QAwlxCmMoNBriB4j4yJMbJ8fABfclAA)

Reply:
ahh ok that makes sense - its because the newline is getting added in the heredoc node rather than here. I think we need to invert that logic, i'll try poking around some more

Reply:
👌 should be good now